src/stores/sounds.js

```diff
@@ -25,7 +26,7 @@ const shouldPlaySounds = hasUserAccount
  * Preferred version is the .ogg, with .flac fallback if .ogg is not supported (Safari)
  */
 const fileExtension = new Audio().canPlayType('audio/ogg') ? '.ogg' : '.flac'
-const isAudioOutputSelectSupported = !!(new Audio().setSinkId)
+const isAudioOutputSelectSupported = !isSafari && !!(new Audio().setSinkId)
```

Comment:
Shall we check for AudioContext.setSinkId instead?

Reply:
That was my first thought
But we don't support `AudioContext.setSinkId` in the code, so atm it's better to go that way

Reply:
I'm afraid that one day Safari will support it, so `isSafari` check won't work